docs/HISTORY.txt

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+1.0.9.4 (unreleased)
```

Comment:
The version here is wrong, due to something really not obvious: We forked this package and changed a lot of things and have not been able to get them merged upstream (not sure anymore if it is due to a lack of time or due to a lack of interest upstream).
Which means that we must for the time being stay on the version from which we forked (1.0.9.3) and add a specific suffix - i.e. 1.0.9.3+perfact.9

Reply:
Do I need to adjust something else for this PR to be merged, so I can create nmu+1 package of perfact-dbutils-zope4 using version of Products.ZPyODBCDA that contains those changes ?